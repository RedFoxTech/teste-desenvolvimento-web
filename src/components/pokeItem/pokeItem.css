.opcao {
    background: rgba(245, 245, 220, 0.62);
    border: 2px solid #000000;
    border-radius: 10px;
    padding: 10px;
}

.opcao:hover {
    background: snow;
}