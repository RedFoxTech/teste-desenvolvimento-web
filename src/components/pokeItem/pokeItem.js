import React from 'react';
import {Link} from 'react-router-dom';
import './pokeItem.css';


const PokeItem = (props) => {
    return (
        <Link className="m-auto opcao d-flex justify-content-center align-items-center" to={props.link} >
            <div className= "m-auto">
                <img className="w-100" src={props.src} alt={props.link}/>
            </div>
        </Link>
    )
}

export default PokeItem;