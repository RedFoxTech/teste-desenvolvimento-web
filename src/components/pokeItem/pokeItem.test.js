import React from 'react';
import pokeItem from './pokeItem';

let wrapper;
beforeEach(() => {
    wrapper = shallow(<pokeItem/>);
})

describe("rendering test/snapshot", () => {
    it('should render a pokeball button', () => {
        expect(wrapper).toMatchSnapshot()
    })
})