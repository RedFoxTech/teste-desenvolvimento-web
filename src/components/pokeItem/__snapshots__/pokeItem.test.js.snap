// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering test/snapshot should render a pokeball button 1`] = `<pokeItem />`;
